apps/shared/avifjpeg.c

```diff
@@ -911,7 +911,7 @@ static avifBool avifJPEGReadInternal(FILE * f,
             iccData = iccDataTmp;
             if (requestedFormat == AVIF_PIXEL_FORMAT_YUV400) {
                 fprintf(stderr,
-                        "The image contains an RGB ICC profile which is incompatible with the requested output "
+                        "The image contains an ICC profile which is incompatible with the requested output "
```

Comment:
Grayscale ICC profiles are also a thing (see iccmaker.h/c). It's true that here we don't actually check the type of profile, just that there is a profile. But I guess it's safe to assume that if the original image was not grayscale, then the icc is not grayscale. We handle grayscale jpegs, right? Is this code path triggered then as well? Because then it probably shouldn't be... I guess this is for another PR if something needs to be fixed.

Reply:
Actually if this is the main goal of this PR then it should be in this PR... (or dropped)

Reply:
> We handle grayscale jpegs, right? Is this code path triggered then as well?

No idea.

> I guess this is for another PR if something needs to be fixed.

Yes.

> Actually if this is the main goal of this PR then it should be in this PR... (or dropped)

No, just "RGB ICC" looked wrong, especially when reading a JPEG.

Reply:
IMO it's a bit misleading to say that ICC is incompatible with grayscale, because fundamentally it's not, but we can say this is meant to say that *we* don't support it...

Reply:
Feel free to send a PR.